### PR TITLE
Django first project

### PR DESCRIPTION
**Pull request by mikescor**

### CODE REVIEW

File: blog/models.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+from django.db import models
+from django.utils import timezone
+
+
+# Create your models here.
+class Post(models.Model):
```
Comment (DrEdi) [inline blog/models.py:6]: Where is docstrings?
Did you use pre-commit for checking pep257?

File: blog/models.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    author = models.ForeignKey('auth.User')
+    title = models.CharField(max_length=200)
+    text = models.TextField()
+
+    created_date = models.DateTimeField(default=timezone.now)
+    published_date = models.DateTimeField(blank=True, null=True)
+
+    def publish(self):
```
Comment (DrEdi) [inline blog/models.py:14]: methods also should have docstings

File: mysite/settings.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+Django settings for mysite project.
+
+Generated by 'django-admin startproject' using Django 1.10.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.10/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.10/ref/settings/
+"""
```
Comment (DrEdi) [inline mysite/settings.py:11]: you don't really need this now. Leave only first string `Django settings for mysite project.`

**Review by DrEdi — CHANGES_REQUESTED** [review]
Please, add views and urls. This is not complete even for first push.

upd: 
everything you did here has no any relation to branch name. You create Django project, but in branch you say that you want to INTEGRATE IT WITH TRAVIS. 

upd2:
also i didn't see in issues that you had to do this. 
Leave this for later and now start to do what you have in issues.
https://github.com/kpi-web-guild/django-girls-blog-mikescor/issues/1
this is your first exercise

**Comment by mikescor** [comment]
thanks for tips.

File: blog/.directory [hunk 4]
```diff
@@ -0,0 +1,4 @@
+[Dolphin]
```
Comment (webknjaz) [inline blog/.directory:1]: It's a bullshit. **Don't ever commit useless files into the repo**

File: .gitignore [hunk 5]
```diff
@@ -81,6 +87,7 @@ celerybeat-schedule
 # virtualenv
 venv/
 ENV/
+myvenv/
```
Comment (webknjaz) [inline .gitignore:90]: Okay, now configure `pyenv` w/ `pyenv-virtualenv` and use it instead

File: .gitignore [hunk 6]
```diff
@@ -6,6 +6,12 @@ __pycache__/
 # C extensions
 *.so
 
+
+
```
Comment (webknjaz) [inline .gitignore:10]: Why did you add so many new lines?

File: blog/migrations/.directory [hunk 7]
```diff
@@ -0,0 +1,4 @@
+[Dolphin]
```
Comment (webknjaz) [inline blog/migrations/.directory:1]: :hankey:

File: mysite/.directory [hunk 8]
```diff
@@ -0,0 +1,4 @@
+[Dolphin]
```
Comment (webknjaz) [inline mysite/.directory:1]: :hankey:

**Review by webknjaz — CHANGES_REQUESTED** [review]
Let's postpone this PR until automatic linters checking configured.

File: blog/admin.py [hunk 9]
```diff
@@ -0,0 +1,4 @@
+from django.contrib import admin
+from .models import Post
```
Comment (MaxymVlasov) [inline blog/admin.py:2]: What your `<app>` name in template `from <app>.models import <ModelName>`?

Comment (webknjaz) [reply blog/admin.py:2]: @MaxymVlasov Does it matter?

**Pull request closed without merging**